content/en/docs/ops/configuration/extensibility/wasm-module-distribution/index.md

```diff
@@ -0,0 +1,143 @@
+{{< /text >}}
+
+The second `EnvoyFilter` provides configuration for the filter, which is an `EXTENSION_CONFIG` patch and will be distributed to the proxy as an Envoy [`Extension Configuration`](https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/extension) (ECDS) resource.
+Once this update reaches the Istio agent, the agent will download the Wasm module and store it in the local file system.
+If the download fails, the agent will reject the `Extension Configuration` update to prevent invalid Wasm filter configuration from reaching the Envoy proxy.
+Because of this protection, with the initial fetch timeout being set to 0, the listener update will not become effective and an invalid Wasm filter will not disturb traffic.
+The important parts of this configuration are:
+
+- Wasm `vm` configuration which points to a remote Wasm module.
+- Wasm extension configuration, which is a JSON string that is consumed by the Wasm extension.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+ name: basic-auth-config
+ namespace: istio-system
+spec:
+ configPatches:
+ - applyTo: EXTENSION_CONFIG
+   match:
+     context: GATEWAY
+   patch:
+     operation: ADD
+     value:
+       name: istio.basic_auth
+       typed_config:
+         "@type": type.googleapis.com/udpa.type.v1.TypedStruct
+         type_url: type.googleapis.com/envoy.extensions.filters.http.wasm.v3.Wasm
+         value:
+           config:
+             vm_config:
+               vm_id: basic-auth
+               runtime: envoy.wasm.runtime.v8
+               code:
+                 remote:
+                   http_uri:
+                     uri: https://github.com/istio-ecosystem/wasm-extensions/releases/download/{{< istio_version >}}.0/basic-auth.wasm
+                   # Optional: specifying sha256 checksum will let istio agent verify the checksum of downloaded artifacts. Missing
+                   # checksum will cause the Wasm module to be downloaded repeatedly.
+                   # To compute the sha256 checksum of a Wasm module, download the module and run `sha256sum` command with it.
+                   # sha256: <WASM-MODULE-SHA>
+             # The configuration for the Wasm extension itself
+             configuration:
+               '@type': type.googleapis.com/google.protobuf.StringValue
+               value: |
+                 {
+                   "basic_auth_rules": [
+                     {
+                       "prefix": "/productpage",
+                       "request_methods":[ "GET", "POST" ],
+                       "credentials":[ "ok:test", "YWRtaW4zOmFkbWluMw==" ]
+                     }
+                   ]
+                 }
+{{< /text >}}
+
+The Istio agent will only intercept and download remote Wasm modules configured via ECDS resources.
+To disable ECDS interception and Wasm downloading in the Istio agent, set the `ISTIO_AGENT_ENABLE_WASM_REMOTE_LOAD_CONVERSION` environment variable to `false`.
```

Comment:
```suggestion
This feature is enabled by default in Istio 1.9. To disable ECDS interception and Wasm downloading in the Istio agent, set the `ISTIO_AGENT_ENABLE_WASM_REMOTE_LOAD_CONVERSION` environment variable to `false`.
```

Reply:
These are the 1.9 docs, so I would leave off the `in Istio 1.9`. It could just say `This feature is enabled by default. To dsable ECDS ...`